src/local_scheduler/local_scheduler.cc

```diff
@@ -728,9 +831,15 @@ void process_message(event_loop *loop,
   case MessageType_GetTask: {
```

Comment:
This makes me think that we actually need to make actor methods *completely* ignore resources for now. The problem is, for example, if actor methods can acquire CPU resources, then when an actor method finishes, it has to trigger the dispatch of tasks, which we aren't doing. When we do make actor methods take resources into account, then we can't distinguish between dispatching actor methods and dispatching tasks, any time we do one, we need to do both.